backend/src/Core/FurnitureShop.Core.Contracts/Web/Products/UpdateProduct.cs

```diff
@@ -1,14 +1,14 @@
 using System;
 using LeanCode.CQRS;
 using LeanCode.CQRS.Security;
+using FurnitureShop.Core.Contracts.Dtos;
 
-namespace FurnitureShop.Core.Contracts.Mobile.Products
+namespace FurnitureShop.Core.Contracts.Web.Products
 {
     [AuthorizeWhenHasAnyOf(Auth.Roles.Admin)]
     public class UpdateProduct : IRemoteCommand
     {
-        public Guid Id { get; set; }
-        public ProductDetailsDTO ProductDetails { get; set; }
+        public ProductWithDetailsDTO ProductDetails { get; set; }
```

Comment:
If you will change the way of providing data in commands, then be consistent in other files